### PR TITLE
About me section and recent works section completed

### PR DESCRIPTION
**Pull request by admirerbrown**
i completed all project requirements on my portfolio project. I performed the following tasks:
-built the full html sections for the project
-built the style-sheet for the project
-fixed all linter errors and cleaned the project
-ensured finish code output looks almost like the figma template.
- implemented code review corrections
- implemented second review suggestions

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -52,13 +53,248 @@ a {
...
+  margin: 20px 25px;
+}
+
+.grid-items {
+  width: 320px;
+  height: 474px;
+  background-color: #ebf0ee;
+  border: #cfd0d1 solid 1px;
+  border-radius: 8px 8px 8px 8px;
+}
```
Comment (mckent05) [inline style.css:74]: - According to the project requirements, the content of each of your project cards should be aligned with flexbox. Please try to fix this 👍 👍

File: index.html [hunk 2]
```diff
@@ -1,64 +1,276 @@
...
-              <a href="https://github.com/admirerbrown/portfolio"
-                ><i class="fa-brands fa-github"></i
-              ></a>
+              <button class="btn-btn" type="button">Ruby on rails</button>
             </li>
+            <li><button class="btn-btn" type="button">Css</button></li>
+            <li><button class="btn-btn" type="button">Javascript</button></li>
+            <li><button class="btn-btn" type="button">Html</button></li>
+          </ul>
+          <a class="view"><button class="btn" type="button">See Project</button></a>
```
Comment (mckent05) [inline index.html]: - To help improve the HTML structure of your page, I suggest you wrap this in either an `a` tag or `button` tag and not both. Please fix this for all project buttons as they might apply 🙏

File: index.html [hunk 3]
```diff
@@ -1,64 +1,276 @@
...
+  <section id="about-me">
+    <div class="about">
+      <ul class="list about-li">
+        <li class="ab-li-1">
+          <h1>About me</h1>
+        </li>
+        <li class="">
+          <img class="side1" src="images/side.svg" alt="side-image" />
+        </li>
+      </ul>
```
Comment (mckent05) [inline index.html]: - To help improve your page's HTML structure, I suggest you do not need to use a `ul` tag to wrap the content here. Please remove the `ul` tag and corresponding `li` tags 🙏

File: index.html [hunk 4]
```diff
@@ -1,64 +1,276 @@
...
+      </p>
+
+      <ul class="list about-li">
+        <li class="">
+          <img class="" src="images/Frame-1.svg" alt="side-image" />
+        </li>
+        <li class="">
+          <a><button class="btn ab-btn-btn" type="button">
+              Get my Resume
+            </button></a>
```
Comment (mckent05) [inline index.html]: - Just as I suggested earlier, please use either an `a` tag or `button` tag, not both. Please fix this 👍 👍

File: index.html [hunk 5]
```diff
@@ -1,64 +1,276 @@
...
+        </li>
+        <li><button class="btn-btn ab-btn" type="button">Github</button></li>
+        <li><button class="btn-btn ab-btn" type="button">codepen</button></li>
+        <li><button class="btn-btn ab-btn" type="button">Gitlab</button></li>
+        <li>
+          <button class="btn-btn ab-btn" type="button">Terminal</button>
+        </li>
+      </ul>
+    </div>
+  </section>
```
Comment (mckent05) [inline index.html]: - To help improve your HTML structure and ensure your page is readable and maintainable. I suggest you include this section in your `about me` section (the section above). Also, you can consider wrapping this section in a `ul` tag and then wrap individual language, frameworks and skills in a `li` tag. Please fix this 👍 👍

File: index.html [hunk 6]
```diff
@@ -1,64 +1,276 @@
...
+        </li>
+        <li>
+          <a href="https://twitter.com/brown_admirer"><i class="fa-brands fa-twitter ab-soc"></i></a>
+        </li>
+      </ul>
+      <a class="ab-mail">
+        <h4>mr.kyere@gmail.com</h4>
+      </a>
+    </div>
+  </section>
```
Comment (mckent05) [inline index.html]: - This section of the page is not meant for this milestone, I advise that you remove this section and include it when you get to the required milestone. Please fix this 🙏 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @admirerbrown,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing correctly ✔️ ✔️ 
- Your portfolio looks good 👍 👍 
- GitHub flow was used correctly 👏 

## Required Changes ♻️
- Your portfolio currently doesn't fit well into all mobile screen sizes i.e it is not responsive across screen sizes `<768px`. I advise that you try to make your portfolio responsive. Please fix this 👍 👍 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -1,64 +1,279 @@
...
         </div>
       </div>
+    </div>
+  </section>
+
+  <!-- recent About-me section -->
+  <section id="about-me">
+    <div class="about">
+      <div class="about-li">
+        <a>
```
Comment (tufoinnkuo10) [inline index.html]: - [optional] the about me is a heading and does not need the `a tag `

File: index.html [hunk 8]
```diff
@@ -1,64 +1,279 @@
...
+      </p>
+
+      <div class="about-li">
+        <img class="" src="images/Frame-1.svg" alt="side-image" />
+
+        <button class="btn ab-btn-btn" type="button">Get my Resume</button>
+      </div>
+    </div>
+
+    <section id="abt-me-items">
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] like the previous reviewer noted, the about me items should be inside the about me `section tag` not on their own personal section tag. They all make up the skills and framework about you and should be together. please fix that.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @admirerbrown 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @admirerbrown , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] All linter checks are passing :white_check_mark: 
- [ ] Implemented all the changes from the previous reviewer :white_check_mark: 

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**